### DISCUSSION TITLE
v1.11.1 - BaseUrl Hotfix

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-desktop#685] [General]
This is a hotfix which fixes the previous release so it can work on servers which are running on a subpath/base url (i.e. `example.com/jellyfin` instead of `jellyfin.example.com`).

Note: It is normal to have to log into your servers again after installing this update if you are upgrading from v1.10.1 or older.

About Windows Downloads:
 - The desktop installer requires administrator rights.
 - You can avoid warnings on files by unblocking the file in the properties dialog.
 - You can manually build a release [using these instructions](https://github.com/jellyfin/jellyfin-media-player#building-for-windows).

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.11.1'>v1.11.1 - BaseUrl Hotfix</a>.</em>

**Comment by bo-hmpl** [comment]
The Windows version ist still broken for me. I can log in now with `IP:port `but when trying to play a video the screen stays blank. Only thing I can do then is closing the window. I see no error and no log entry that points to a problem.

**Reply by SwaggyMacro** [reply]
same

**Reply by SwaggyMacro** [reply]
I have upgraded the server version to 10.9.6, and it works.

**Reply by SwaggyMacro** [reply]
but it will not responding sometimes when I upgrade the server version to 10.9.6, including 1.10.0 1.10.1 1.11.1.
and It only plays audio and the screen stays black while playing video.
![1717910871409](https://github.com/jellyfin/jellyfin-media-player/assets/38845682/11fb7ce0-9629-494b-916a-006683e0f72e)

**Comment by shoaib42** [comment]
Using mac here, after pointing to the server (with a basePath) the client is stuck at the logo. 
```
Server version 10.9.6
Web version 10.9.6
Build version 10.9.6
```
<img width="539" alt="image" src="https://github.com/jellyfin/jellyfin-media-player/assets/35446260/d627deea-db01-4625-bfee-0b94e1a1ed2d">

**Reply by mvdschoot** [reply]
Same for me

**Comment by Kaschtelator** [comment]
![image](https://github.com/jellyfin/jellyfin-media-player/assets/61439673/ffa04afe-d062-4e8a-bee6-9d18bc15987e)

**Please for God's sake stop releasing updates, I'm sick of constant errors!**

**Comment by whayle** [comment]
I'm prompted to upgrade again when running this update (jellyfin-media-player_1.11.1-1_amd64-jammy.deb) , and the app shows as 1.10.1 in the dashboard.

**Comment by elkenders** [comment]
I'm running the latest server instance (10.9.6 through Docker). When I update to this v1.11.1 release on Silicon mac it crashes imediately every time. I've tried clearing cache on the mac client. 1.10.1 seems to work fine still.

**Comment by whayle** [comment]
This release also no longer plays .dsf audio as in previous releases

**Comment by BearPawGames** [comment]
Subtitles have an issue on this version. On the start of an episode subs appear in the lower left corner only and each letter is overlayed on itself.
![JellyfinSubIssue](https://github.com/jellyfin/jellyfin-media-player/assets/148607086/042d7589-abd7-42db-8af1-38a8694887c5)

**Reply by FlaireBare** [reply]
Edit: I also got this on Ver 1.10.0 too :( 

I have to turn off subtitles, and then turn them back on again every time I watch something to make them work. I've reverted back to Ver 1.10.0 as in every version after, this happens. Please Fix Jellyfin Crew <3 ^_^

**Reply by LewisJustin** [reply]
If it helps the team find the issue, I discovered that this will happen every time I use the "next episode" button, but when I just sit back and wait for the next episode to start on its own there are no issues.

**Reply by dcquence** [reply]
Looks like replacing libmpv-2.dll is still the only fix for this.
Grab [mpv-dev-x86_64-20240414-git-6a8b130.7z](https://sourceforge.net/projects/mpv-player-windows/files/libmpv/mpv-dev-x86_64-20240414-git-6a8b130.7z/download)
Put the libmpv-2.dll from the archive into C:\Program Files\Jellyfin\Jellyfin Media Player

**Comment by gjpc** [comment]
Updated the server to 10.9.7 on June 25 and Windows client 1.11.1 is broken. I can only play media on the home page.  There is no media category navigation. Once I try to navigate media info, the app appears to hang. Selecting play media on the home page only plays the audio once the hang occurs.  Clicking on any title to see the media information causes the hang. Search works but only to play media not to view media information.

Navigation works on the firetv device.

The only thing in the log when the navigation log hang occurs is:

  _[2024-06-25 16:03:08.305 -07:00] [INF] WS "10.10.0.19" request
  [2024-06-25 16:04:37.614 -07:00] [INF] WS "10.10.0.19" closed
  [2024-06-25 16:04:43.615 -07:00] [INF] WS "10.10.0.19" request_

**Reply by IonicFalcon** [reply]
Can confirm same thing is happening for me on both macOS and Linux clients. Does seem to be client issue rather than server as the Android TV app works fine and downgrading my client to 1.10.0 also plays perfectly

**Comment by Coding4Peanuts** [comment]
When switching between servers, home page stays on the last server's home page instead of switching to new one. I'm on 1.11.1 and using an 2019 intel Macbook Pro

**Comment by lukaslindnermusic** [comment]
Unfortunately, the app still crashes with the Apple Silicon native build on my Macbook Air M1. :/ 
I have to use the Intel build running with Rosetta on that device.

**Reply by lukaslindnermusic** [reply]
it crashes directly on start up, no window appears. The app icon jumps two times or so and then it stops and a crash error gets displayed by macOS.

**Comment by MChrisp** [comment]
After the upgrade (and also a full reinstall) after selecting the server (subdomain) I get a blank screen. This was not the case with a 1.10 version. (Win 11 on Intel)

**Comment by bluttmer** [comment]
I just upgraded both server and player and the player does load, its just a blank black screen...

Windows 10 x64

Server version - 10.9.10
Web version - 10.9.10
Build version - 10.9.10

What steps can I do.

**Reply by gjpc** [reply]
Server version 10.9.9
Web version 10.9.1
Build version 10.9.1

Work. Test all upgrades on non production machines.

**Comment by reggie299** [comment]
hi. windows user and when tryin to just connect the server i get error saying unable to connect right now

**Comment by Sammac01** [comment]
I am on a Mac. I downloaded this version, but it will not accept my server once installed. I type the server URL, and it will not allow me to move forward; I can't even click the connect button. I downloaded Intel for 1.10.1, and my server URL works fine. Is there a way to fix this for 1.11.1?

**Comment by ALegitCookie** [comment]
M2 Pro Macbook Pro on Ventura 13.6
Jellyfin Media Player crashes immediately upon opening after installing this version.
[](url)
<img width="264" alt="Screenshot 2024-12-04 at 1 47 39 PM" src="https://github.com/user-attachments/assets/f33d028f-d8ef-4216-a8ec-1429ef230a0d">

**Reply by q2go** [reply]
Our copy of macOS is too old 🙁 (I am also running Ventura 13.7.2):
```
$ /Applications/Jellyfin\ Media\ Player.app/Contents/MacOS/Jellyfin\ Media\ Player
dyld[6763]: Symbol not found: _$sSo13NSWindowLevela6AppKit01_cdD23NumericRawRepresentableACMc
  Referenced from: <4FD5D214-DDF0-302C-9158-A1BE7E5BD317> /Applications/Jellyfin Media Player.app/Contents/Frameworks/libmpv.2.dylib (built for macOS 14.0 which is newer than running OS)
  Expected in:     <8F65E267-53FB-35CE-92EC-F87C29886082> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
Abort trap: 6
```

**Comment by Kavey78** [comment]
I've been developing some interesting issues with this package recently.  On debian bookworm it usually runs great when I first start it.  Occasionally it'll hang in the background and I need to kill and restart it, but that's not very frequent.  The most frequent problem has to do with inactivity.  If I were to pause a video for an extended period of time it will most likely not resume properly.  The level of how it's not proper can be different and not sure if it's related to how long it's been paused.  Sometimes the video is choppy and discolored other times audio is completely missing, and other times it's just frozen until I restart the app.  Less frequently, but simply leaving it on the main screen can result in an app that doesn't function until I kill an restart the processes.  But if I am actively watching shows and not pausing video for too long I have no issues.

I have similar issues with trixie on other systems just to see if it was related to the bookworm version or not.  I do not experience similar problems with other clients. Android, Roku or even browsers.
